Week3/home-work/app.js

```diff
@@ -0,0 +1,79 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const app = express();
+const queries = require('./query.js');
+
+app.use(bodyParser.json());
+
+
+
+app.get('/user/:id', function (req, res) {
```

Comment:
The client usually dont know the id of the next user it will create. That is why it is a good idea to have some AUTO INCREMENT in the user table too. =)